content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md

```diff
@@ -0,0 +1,100 @@
+
+* Review your client and server service settings. 
+  Note all certificate, device type, port, protocol, TLS crypt authentication, and additional parameter settings. 
+
+## Installing the WG-Easy Application 
+
+After disabling the OpenVPN services, install the **wg-easy** application. 
+Go to **Apps** click on **Available Applications** and locate the **wg-easy** application widget.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyAppWidget.png" alt="wg-easy Application Widget" id="1: wg-easy Application Widget" >}}
+
+Click **Install** to open the **wg-easy** configuration wizard.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyApplicationName.png" alt="Install wg-easy Application Name and Version" id="2: Install wg-easy Application Name and Version" >}}
+
+Accept the default value or enter a name in **Application Name**.
+
+Enter the configuration settings. 
+Enter the public host name or IP of your VPN server in **Hostname or IP**.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyConfigSettings.png" alt="wg-easy Configuration Settings" id="3: wg-easy Configuration Settings >}} 
+
+If you use or want to protect access to the WG-Easy web UI enter a password in **Password for WebUI**.
+
+Accept the default values in **Persistent Keep Alive** and **Clients MTU**, or enter the values you want to use. 
+To change the time the connection remains alive, enter a value in seconds in **Persistent Keep Alive**. When set to zero, connections are not kept alive. 
+
+Accept the default IPs in **Clients IP Address Range** and **Clients DNS Server**, or enter the IP addresses the client to uses.
+
+To specify allowed IP addresses, click **Add** to the right of **Allowed IPs** for each IP address you want to enter. If you do not specify allowed IPs, the application uses 0.0.0.0/0.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyConfigAddAllowIPAndEnvironments.png" alt="wg-easy Allowed IPs and Environments" id="4: wg-easy Allowed IPs and Environments" >}} 
+
+To specify environment variables, click **Add** to the right of **WG-Easy Environment** for each environment variable you want to add. 
+Variables you can add are listed in the table below.
+{{< expand "Environmental Variables" "v" >}}
+{{< truetable >}}
+| Variable | Description |
+|----------|-------------|
+| WD_DEVICE | Enter the interface name or ID for the ethernet device WireGuard traffic should forward through. |
+| WG_PRE_UP | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
+| WG_POST_UP | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
+| WG_PRE_DOWN | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
+| WG_POST_DOWN | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
```

Comment:
Default value is not `...` (The default value is in the file in the link.) probably just remove the last sentence

Reply:
gone in the next commit